lib/tasks/represent_downstream.rake

```diff
@@ -85,6 +85,21 @@ namespace :represent_downstream do
     )
   end
 
+  desc "
+  Represent documents by document type(s) downstream on the high_priority sidekiq
+  queue
+  Usage
+  rake 'represent_downstream:high_priority_document_type[organisation]'
+  "
+
+  task :high_priority_document_type, [:document_types] => :environment do |_t, args|
```

Comment:
The code looks good! Could we move this task into the `:high_priority` namespace below and change the name to just `document_type`? Then it'll be called with `rake represent_downstream:high_priority:document_type[...]`.

Reply:
Thanks @thomasleese! I have updated this now 😄